pkg/health/provider.go

```diff
@@ -0,0 +1,64 @@
+package health
+
+import (
+	"sync"
+)
+
+// Attr is a key-value pair that can be attached to a health check
+type Attr struct {
+	Key   string
+	Value string
+}
+
+// StrAttr creates a new string attribute
+func StrAttr(key, value string) Attr {
+	return Attr{Key: key, Value: value}
+}
+
+// ReportOK reports that a check was successful
+func ReportOK(check Check, attributes ...Attr) {
```

Comment:
Is the reason we didn't use https://pkg.go.dev/go.opentelemetry.io/otel@v1.25.0/attribute because we're afraid of future incompatible changes?

Reply:
just not to entangle the interface and implementation together. 
also, theoretically, there might be some other kind of health check reporter for OSS or Enterprise in the future.